cmd/podman/images/build.go

```diff
@@ -191,6 +191,7 @@ func buildFlags(cmd *cobra.Command) {
 		_ = flags.MarkHidden("compress")
 		_ = flags.MarkHidden("volume")
 		_ = flags.MarkHidden("output")
+		_ = flags.MarkHidden("logsplit")
```

Comment:
I think we should not hide since it's mentioned already in the Podman man pages.

Reply:
This is for podman -remote.

Reply:
Ah, thanks!